crates/bindings-sys/src/lib.rs

```diff
@@ -355,6 +380,11 @@ pub mod raw {
     #[repr(transparent)]
     pub struct RowIter(u32);
 
+    impl RowIter {
+        /// An invalid handle, used e.g., when the iterator has been exhausted.
+        pub const INVALID: Self = Self(0);
+    }
```

Comment:
Would it make more sense to have RowIter be a `NonZeroU32`, and use `Option<RowIter>` to represent `INVALID`?